core/data_category.js

```diff
@@ -172,6 +172,20 @@ Blockly.DataCategory.addHideVariable = function(xmlList, variable) {
 };
 
 /**
+ * Construct and add a data_listcontents block to xmlList.
+ * @param {!Array.<!Element>} xmlList Array of XML block elements.
+ * @param {?Blockly.VariableModel} variable Variable to select in the field.
+ */
+Blockly.DataCategory.addDataList = function(xmlList, variable) {
```

Comment:
Consider renaming some of the references to "variable" to make it clear it's a list.